pico/client.py

```diff
@@ -120,7 +122,12 @@ def clear_auth(self):
         self.session.headers.pop('Authorization', None)
 
 
-def load(module_url):
-    url, module_name = module_url.rsplit('/', 1)
-    client = PicoClient(url)
+def load(module_name):
+    ip = os.getenv("PICO_IP", '127.0.0.1')
```

Comment:
i'd suggest you to follow repo owner's syntax while using quotes. In this repo single quote were used. @jezscha